installer/installer.go

```diff
@@ -66,5 +69,18 @@ func NewInstaller(ctx context.Context, osDist, arch, k8sVersion string, download
 	osbundle := reg.ResolveOsToOsBundle(osArch)
 	addrs := downloader.GetBundleAddr(osbundle, k8sVersion)
 
-	return algo.NewUbuntu20_04Installer(ctx, arch, addrs)
+	// Use appropriate installer based on OS version
+	var installer K8sInstaller
+	var err error
+
+	if strings.Contains(osbundle, "Ubuntu_22.04") {
+		installer, err = algo.NewUbuntu22_04Installer(ctx, arch, addrs)
+	} else {
+		installer, err = algo.NewUbuntu20_04Installer(ctx, arch, addrs)
```

Comment:
Should we even support Ubuntu 20.04 anymore? The rest of PCD does not.

Reply:
May be we might not use it in PCD. But we might need to have it for open source users?